exercises/two-fer/.meta/solutions/two_fer.rb

```diff
@@ -0,0 +1,13 @@
+module BookKeeping
+  VERSION = 1
+end
+
+class TwoFer
+  def self.two_fer(name=nil)
+    if name == nil
+      return "One for you, one for me."
+    else
+      return "One for #{name}, one for me."
+    end
+  end
```

Comment:
I wonder if we can write more idiomatic ruby for this. For example you don't need ```return```. 
Ruby method will return the last evaluated value by default. So the code above is the same as:
```
def self.two_for(name = nil)
   if name == nil
      "One for you, one for me"
  else 
     "One for #{name}, one for me"
  end 
end
```

But! Ruby is so awesome and expressive - you don't even need to check for nil explicitly,  nil, empty string, false is ```falsy``` in ruby, conversely any other values are ```truthy```.  
```
if !name 
    "One for you, one for me"
else
    "One for #{name}, one for me"
end
```

```if !condition``` statement usually is written as ```unless condition``` in ruby too 

Sorry is a long winded comment - this is just style comment of course - your code is correct functionally. 

I am not sure if exercism worry about idiomatic ruby - what do you think @Insti ?

Reply:
I'm currently not too concerned about having idiomatic Ruby in the example solution, as long as it passes the tests.

Your suggestions above are certainly valid.

If I were solving this I would eliminate the conditional completely and write it as:

```ruby
def self.two_for(name = 'you')
  "One for #{name}, one for me"
end
```

The best place to discuss this sort of thing is on the Exercism site itself. 😄